### ISSUE TITLE
Node Version

### ISSUE THREAD

**Issue by pfulton** [sparkbox/carbon#18]
Should we specify a minimum Node version in our `package.json`?

**Comment by iAmNathanJ** [comment]
Yes, I think we should specify the current LTS version is in the engines field. Should we also consider adding the actual node package to our deps? https://www.npmjs.com/package/node
